main.py

```diff
@@ -1,10 +1,56 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    Race.objects.all().delete()
+    Skill.objects.all().delete()
+    Guild.objects.all().delete()
+    Player.objects.all().delete()
+
+    with open("players.json") as f:
+        players = json.load(f)
+        for player in players:
+            race_name = players[player]["race"]["name"]
+            race_description = players[player]["race"]["description"]
+            race, create = Race.objects.get_or_create(
+                name=race_name,
+                description=race_description
+            )
+
+            for skill in players[player]["race"]["skills"]:
+                name = skill["name"]
+                bonus = skill["bonus"]
+                Skill.objects.get_or_create(
+                    name=name,
+                    bonus=bonus,
+                    race_id=race.id
+                )
+
+            if players[player]["guild"] is not None:
+                guild_name = players[player]["guild"]["name"]
+                guild_description = players[player]["guild"]["description"]
+                guild, create = Guild.objects.get_or_create(
+                    name=guild_name,
+                    description=guild_description
+                )
+                guild_id = guild.id
+            else:
+                guild_id = None
```

Comment:
You could make this part more efficient using walrus operator
```suggestion
            if (guild_data := players[player]["guild"]):
                guild, _ = Guild.objects.get_or_create(
                    name=guild_data ["name"],
                    description=guild_data ["description"]
                )
                guild_id = guild.id
            else:
                guild_id = None
```

Reply:
не работает код если я так использую оператор walrus

Reply:
код не работает если я так сделаю с оператором walrus